.github/workflows/publish.yml

```diff
@@ -105,4 +107,4 @@ jobs:
       - name: Install package
         shell: bash -el {0}
         run: |
-          conda install -c mindee doctr
+          conda install -c mindee -c conda-forge -c pypdfium2-team -c bblanchon doctr
```

Comment:
@frgfm @mara004 the channels was missing before correct ? Or does it use it from the config and we can keep only the mindee channel ?

Reply:
changed to:

```
conda config --set channel_priority strict
conda install -c mindee -c pypdfium2-team -c bblanchon -c conda-forge python-doctr
```